src/Link.js

```diff
@@ -19,7 +19,7 @@ import {navigate, getBasepath} from "./router";
  */
 export const setLinkProps = (props) => {
 	const onClick = (e) => {
-		if (!e.shiftKey && !e.ctrlKey && !e.altKey) {
+		if (!e.shiftKey && !e.ctrlKey && !e.altKey && props.target !== "_blank")) {
```

Comment:
@tugceakin Seems like you accidentally added an extra closing parantheses here `)`.

@Paratron I have a PR that also fixes this: https://github.com/Paratron/hookrouter/pull/152

Reply:
@Paratron @prageeth the PR I put in #148 already fixes this - all it needs is to be merged by @Paratron :)

Reply:
@ogazitt Sorry I should have looked around. I only noticed it when i was fixing my issue with the ForwardRefs in Material UI.